travis/test.sh

```diff
@@ -15,9 +15,10 @@ flutter analyze --flutter-repo
 (cd packages/flx; dart -c test/all.dart)
 
 (cd dev/manual_tests; flutter test)
+(cd examples/calculator; flutter test)
+(cd examples/flutter_gallery; flutter test)
 (cd examples/hello_world; flutter test)
 (cd examples/layers; flutter test)
-(cd examples/flutter_gallery; flutter test)
```

Comment:
please leave flutter_gallery near the end, because (when the main test in there isn't disabled) it's one of the longest-running tests.